test/unit/edgeware/enricher.spec.ts

```diff
@@ -49,6 +54,31 @@ const api = constructFakeApi({
       ],
     }
   },
+  'validators.at' : async (hash) => {
```

Comment:
Regarding tests, if you wanted to verify the difference in calls between v31 and v38, you could have two "known" hashes, each corresponding to one of the versions, and then return different results depending which you receive.